rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
 channel = "nightly-2023-03-15"
 components = ["clippy", "llvm-tools-preview", "rustfmt"]
-targets = []
+targets = ["wasm32-unknown-unknown"]
```

Comment:
Don't think we use wasm.

Reply:
Removed. I thought these actors need to be compiled into wasm and was about to create a CI job for that

Reply:
All the WASM/js stuff is limited to the builtin-actors. Our crates are entirely outside of that ecosystem.